src/App.vue

```diff
@@ -364,23 +331,23 @@ export default class App extends Vue {
   /** Data for fee summary component. */
   get feeFilingData (): Array<FilingDataIF> {
     let filingData = [] as Array<FilingDataIF>
+
     if (this.getFilingData) {
       filingData = cloneDeep(this.getFilingData)
-      if (this.isTypeCoop && this.getFilingData.length > 0) {
-        // Only set Future Effective and Priority to Special Resolution Fee
-        const specialResolutionFilingData = filingData.find(x => x.filingTypeCode === FilingCodes.SPECIAL_RESOLUTION)
-        if (specialResolutionFilingData) {
```

Comment:
In this old code, this logic is executed for ALL coop filings (per line 369).

`specialResolutionFilingData` only exists for [dissolution filings](https://github.com/bcgov/business-create-ui/blob/a689f82a319d7c16134eb4a31002efacbae2803e/src/resources/Dissolution/CP.ts#L21).

This means that non-dissolution coop filings (IAs) didn't run this code block, so they didn't update the filing data flags... fortunately we did not (yet) implement staff payment for coop IAs, so this didn't break.

Anyway, the new code is only for coop dissolution filings, so coop IAs will use the fallback code on new line 349 (and therefore work when we implement staff payment).

Also, the new code checks for the `specialResolutionFilingData` object on new line 344 (so we don't get an exception), but otherwise it works correctly and safely.

Coop dissolution fees before any staff payment options selected:
![image](https://user-images.githubusercontent.com/10002889/213027793-11699258-92a0-4ab1-ac44-77dda89bc96e.png)

Coop dissolution fees with No Fee option selected:
![image](https://user-images.githubusercontent.com/10002889/213027901-5803b78c-b6a2-4013-9709-172ba4983a2b.png)

Coop dissolution fees with a payment option and Priority checked:
![image](https://user-images.githubusercontent.com/10002889/213028005-dc4a830b-3e93-44a1-ad78-30a559199d1a.png)

As for why I touched this, it's because I had to deal with Full vs Limited restorations, which implement a similar array for full vs limited fees (and a subtitle). So I cleaned up this method, and then I ended up implementing full vs limited a different way.